admin_interface/templates/admin/change_list.html

```diff
@@ -18,7 +20,12 @@ <h3 id="changelist-filter-clear">
                {# Translators: don't translate this, the django catalog already contains it #}
                <a href="{{ cl.clear_all_filters_qs }}">&#10006; {% translate "Clear all filters" %}</a>
              </h3>
-            {% endif %}
+             {% endif %}
+            {% elif active_date_hierarchy and list_filter_removal_links %}
+             {% admin_interface_date_hierarchy_removal_link cl active_date_hierarchy %}
+             <h3 id="changelist-filter-clear">
+               <a href="{{ cl.clear_all_filters_qs }}">{% translate "Clear all filters" %} &#10006;</a>
+             </h3>
             {% endif %}
             {% for spec in cl.filter_specs %}{% admin_list_filter cl spec %}{% endfor %}
           </div>
```

Comment:
To validate, I have checked combinations of:
- option removal links false: only see `x Clear all filters` when a filter is active
- option removal links true, have one or more filters active: see `Filter 1: value x` and `Clear all filters x`
- option true, have date hierarchy active: see `Date filter: date x` and `Clear all fitlers x`
- option true, have filter active and date hierarchy active: see `Date filter: date x`, `Filter 1: value x`, `Clear all filters x`

Test date hierarchy with year only, year and month, year month day.

Reply:
Thank you for this info, I would have tested the same combinations by hand.

Reply:
Would be great to have PR deployed somewhere to test all options in a demo project, but nothing is free!

Reply:
True, we could open an issue for that.